appyx-interactions/desktop/src/desktopMain/kotlin/com/bumble/appyx/interactions/desktop/DesktopDispatcher.kt

```diff
@@ -0,0 +1,26 @@
+package com.bumble.appyx.interactions.desktop
+
+import kotlinx.coroutines.InternalCoroutinesApi
+import kotlinx.coroutines.MainCoroutineDispatcher
+import kotlinx.coroutines.Runnable
+import kotlinx.coroutines.internal.MainDispatcherFactory
+import kotlinx.coroutines.runBlocking
+import kotlin.coroutines.CoroutineContext
+
+@OptIn(InternalCoroutinesApi::class)
+internal class DesktopDispatcherFactory : MainDispatcherFactory {
```

Comment:
can we use https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-javafx/ instead?

Reply:
I tried and failed. I'll try with swing one.